### DISCUSSION TITLE
Access the raspberry pi integrated into the robot

### DISCUSSION THREAD

**Discussion by Juancams** [iRobotEducation/create3_docs#449] [Questions]
### How are you connecting to your Create 3?

Wi-Fi and USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Jetson AGX Xavier / Ubuntu 20 / ROS 2 Foxy

### Which version of ROS 2 is installed on your computer?

Other

### Which firmware version is installed on your robot?

 G.3.2

### Which RMW is your robot running?

CycloneDDS

### Is the robot connected to a network? If so, what is the network type?

Home network

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

No

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

Hello, I would like to know if there is any way to access the integrated computer (Raspberry Pi 4 as far as I understand), and know if port C can be used to connect the sensors directly to it.

Thank you very much in advance^^

**Comment by shamlian** [comment]
If you have a Create 3 robot, there is no integrated user-programmable single board computer, though you could [add one](https://iroboteducation.github.io/create3_docs/hw/rpi_hookup/). If you have a derivative product (like Turtlebot 4), you should check that product's [documentation site](https://turtlebot.github.io/turtlebot4-user-manual/) or [issue tracker](https://github.com/turtlebot/turtlebot4/issues/).

**Reply by Juancams** [reply]
I thought a raspberry pi4 was already integrated. 

Thank you very much for your prompt response.
